tools/tasks/seed/build.js.prod.ts

```diff
@@ -36,14 +36,4 @@ function buildTS() {
   return result.js
     .pipe(plugins.template(templateLocals()))
     .pipe(gulp.dest(TMP_DIR));
-}
-
-/**
- * Copy template files for the production environment if in LAZY TEMPLATE mode.
- */
-function copyTemplates() {
```

Comment:
Hi @vyakymenko, just a question: How is `copyTemplates()` not used? It gets called in line 49?